src/openrct2/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ add_library(${PROJECT_NAME} ${OPENRCT2_CORE_SOURCES} ${OPENRCT2_CORE_MM_SOURCES}
 set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "")
 SET_CHECK_CXX_FLAGS(${PROJECT_NAME})
 
+# GCC likes us to pass the -lstdc++fs flag to link C++17 filesystem implementation.
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
```suggestion
if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND NOT MINGW)
```

Reply:
Ping @tomlankhorst . I would commit this myself, but have no rights to do so

Reply:
Yes :) Will do now.